docs/current_docs/cookbook/cookbook.mdx

```diff
@@ -162,6 +174,12 @@ When working with private Git repositories, ensure that [SSH authentication is p
 ```typescript file=./snippets/clone-git-repository/typescript/index.ts
 ```
 
+</TabItem>
+<TabItem value="TypeScript (SSH)">
+
+```typescript file=./snippets/clone-git-repository-ssh/typescript/index.ts
+```
+
 </TabItem>
 <TabItem value="PHP">
```

Comment:
@charjr can you please add a PHP equivalent example here?

@grouville this is not a blocker for merge